cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapsTest.java

```diff
@@ -507,4 +507,28 @@ void ContinuousVariable_ScoringTypeError() {
                             "MeasureScoring type: Continuous Variable, is not an accepted Type for care-gaps service"));
         }
     }
+
+    // MinimalProportionResourceBasisSingleGroup
+    @Test
+    void MinimalProportionResourceBasisSingleGroup_Subject() {
```

Comment:
Question:  is the below the part that makes it a non-boolean basis?

```json
  "extension": [ {
    "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-populationBasis",
    "valueCode": "encounter"
  },
```

Reply:
if "basis" is not explicitly a part of a measure profile, so the extension assists in identifying this

1. no extension => assumes "boolean" basis
2. extension with valueCode = "boolean" => "boolean" basis
3. extension with valueCode != "boolean" => "resource" basis or non-boolean

this communicates if CQL results are a collection of resources, or simply a boolean

on a Measure report, this communicates that population.count, is an accumulation of resources or accumulation of boolean=true for the subject